src/status_im/translations/en.cljs

```diff
@@ -602,7 +602,7 @@
    :transactions-filter-type             "Type"
    :transactions-filter-select-all       "Select all"
    :view-transaction-details             "View transaction details"
-   :transaction-description              "Please wait for at least 12 confirmations to make sure your transaction is processed securely"
+   :transaction-description              "Transactions with 12 confirmations can be considered irreversible"
```

Comment:
Seems odd to repeat the same wording in different contexts?

Reply:
Please help me understand the use case here. In previous, my understanding was that it appeared when users have just sent a tx. When does this appear?

Reply:
My reasoning for making them the same, is that any time that idea is expressed, it should be done in a consistent manner. 

You're right, however, that context might call for makign these difference. As is, I don't see a reason. e.g. if the user can take some action or another to affect the transaction